core/app/finders/spree/option_values/find_available.rb

```diff
@@ -0,0 +1,35 @@
+module Spree
+  module OptionValues
+    class FindAvailable
+      def initialize(scope: OptionValue.spree_base_scopes, currency: nil, taxon: nil)
```

Comment:
@mad-eel let's have only 2 arguments for this class `scope` and `product_scop` - in the future we will also scope products by store - after moving this code https://github.com/spree-contrib/spree-multi-domain/blob/master/app/models/spree/store_product.rb into Spree core